### PR TITLE
Disable currently failing wpt

### PR DESCRIPTION
**Pull request by sherryzy**
b/332367155

### CODE REVIEW

**Review by bokssssss — COMMENTED** [review]
sherryzy:disable_wpt

**Comment by bokssssss** [comment]
sherryzy:disable_wpt

File: cobalt/build/cobalt_configuration.py
Comment (bokssssss) [inline cobalt/build/cobalt_configuration.py:1]: cobalt/build/cobalt_configuration.py

**Comment by sherryzy** [comment]
@andrewsavage1 Can you help validate I enabled the tests and those web platform tests are actually being run?

**Review by andrewsavage1 — APPROVED** [review]

**Pull request merged**